portfolionav.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="portfolio.css">
+	<title>Portfolio Page</title>
+</head>
+<body>
+	<nav>
+		 <div class="name">
+	        <ul>
+				<li>Isabelle Cochrane</li>
+			</ul>
+		</div>
+		<div class="navlinks">
+	<ul>
+	    <li><a href=portfolionav.html>Projects</a></li>
+	    <li><a href=aboutme.html>About Me</a></li>
+	    <li><a href=contact.html>Contact</a></li>
+	  </ul>
+	 </div>
+	</nav>
+		<div class="content">
+			<h1>Project Portfolio</h1>
+			<h2>Website 1</h2>
+				<div class="first example">
+					<div class="written_content">
+					<ul>
+						<li>Technologies Used:</li>
+						<li>Ruby on Rails</li>
+						<li>HTML</li>
+						<li>CSS</li>
+					</ul>
+							
+					<p>This is the example for a first bit of porfolio content, please click on the image to investigate the website i have built This is the example for a first bit of porfolio content, please click on the image to investigate the website i have built This is the example for a second bit of porfolio content, please click on the image to investigate the website i have built</p>
+					</div>
+						<a href=contentone.html><img src="content_example.png" alt="Website 1"></a>	
```

Comment:
Missing double quotes around your href here!

Also, rather than linking to a page that doesn't exist I'd suggest that you just link to # like this:

`<a href="#">` 

this basically creates a dummy link